docs/source/plugins/01_quick_start.rst

```diff
@@ -104,22 +96,37 @@ The highlights here are for:
 :myplugin.py: Implementation of the hooks for customization of the UI interface, or the solver configuration hooks.
 :myplugin.c:  Implementation of the hooks for customization of solver.
 
-
 Check out the :ref:`Plugin Structure section <plugin_structure-section>` for more details about how the folder and files are structured, and
 also, check the :ref:`plugin-by-example-section` that shows how to create simple plugins that make use of the `User Interface Hooks` and the `Solver Hooks`.
 
+Invoke support
+--------------
```

Comment:
I would not say "invoke support", because "support" conveys the idea that it is somehow "optional", which in fact is central on how we manage tasks now.

```suggestion
Invoke
-------
```

Reply:
What about `Invoke tasks`? Just to have a more meaningful title.

Reply:
Fixed